html/portfolio.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
if this will be the home page of your portfolio, it should be index.html by default.